stixcore/tmtc/packets.py

```diff
@@ -244,7 +244,8 @@ def __init__(self, data, idb=None):
 
         self.idb = idb
         if not idb:
-            self.idb = self.idb_manager.get_idb(obt=self.data_header.datetime)
+            self.idb = self.idb_manager.get_idb('2.26.35')
+            # self.idb_manager.get_idb(obt=self.data_header.datetime)
```

Comment:
@nicHoch  this was what I meant about it being difficult to override the IDB version

TODO remove before merge